third_party/rust/rand_chacha/v0_2/BUILD.gn

```diff
@@ -37,8 +37,5 @@ cargo_crate("lib") {
     "//brave/third_party/rust/ppv_lite86/v0_2:lib",
     "//brave/third_party/rust/rand_core/v0_5:lib",
   ]
-  features = [
-    "simd",
-    "std",
-  ]
+  features = [ "std" ]
```

Comment:
On t he other hand, this seems incorrect; `simd` is implied by `std` in rand_chacha v0.2.2 although it's deprecated in favour of the `ppv_lite86` feature used in v0.3.

Any ideas here, @bridiver? Seems like this could be a performance regression. I thought `BUILD.gn` had to exhaustively list all features, but e.g. upstream [doesn't enable](https://source.chromium.org/chromium/chromium/src/+/main:third_party/rust/rand_chacha/v0_3/BUILD.gn;l=39) the ppv feature